scripts/extensions/markForUser/src/extension.tsx

```diff
@@ -23,6 +30,23 @@ const extension: IExtension = {
                 globalMenuHorizontal: [getMarkedForMeComponent(superdesk)],
                 articleListItemWidgets: [getDisplayMarkedUserComponent(superdesk)],
                 authoringTopbarWidgets: [getDisplayMarkedUserComponent(superdesk)],
+                personalSpace: {
+                    getSections: () => superdesk.session.getCurrentUser().then((user) => {
+                        const items: Array<IPersonalSpaceSection> = [
+                            {
+                                type: 'markedForMe',
```

Comment:
1. It should work to change the value of the `type` and still have it working. It wouldn't be possible now, because that exact string is [hardcoded in the core](https://github.com/superdesk/superdesk-client-core/pull/3823/files#diff-7ee9b9f524733890c3aca8f73439db650f27c47bb417643a8a8cb8c51aa06a2eR110). You should remove that hardcoding and read the data in registered extensions instead.
2. It would be good to rename `type` to `id` if it doesn't do anything else except identification.